config/jobs/kubernetes/sig-cloud-provider/aws/kops/kops-periodics-versions.yaml

```diff
@@ -19,24 +19,24 @@ periodics:
       - --env=KUBE_SSH_USER=admin
       - --extract=ci/latest-1.18
       - --ginkgo-parallel
-      - --kops-version=https://storage.googleapis.com/kops-ci/bin/latest-ci-updown-green.txt
```

Comment:
this kops version is what is published by this `updown` job here:

https://github.com/kubernetes/test-infra/blob/0d84f4912879c865dae644733c62379c77dcdd76/config/jobs/kubernetes/sig-cloud-provider/aws/kops/kops-periodics.yaml#L219-L247

https://testgrid.k8s.io/sig-cluster-lifecycle-kops#kops-aws-updown

I believe that job just creates and destroys a simple cluster to make sure that kops' master branch isn't totally broken. if it succeeds it publishes that commit to this URL. This way if something goes wrong on master, we don't end up failing all of our periodics because of it. Many of the other jobs in kops-periodics.yaml use updown-green as well, so maybe we should try to be consistent?

Reply:
Updated :)